tests/models/test_NaiveBayes.py

```diff
@@ -0,0 +1,141 @@
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+import scipy.stats
+from pymc3 import summary
+from sklearn.model_selection import train_test_split
+
+from pymc3_models.exc import  PyMC3ModelsError
+from pymc3_models import GaussianNaiveBayes
+
```

Comment:
Extra blank line between the imports and the class per PEP8 (https://www.python.org/dev/peps/pep-0008/#imports).

Reply:
Are you refering to the "2 blank lines before classes" rule?